src/main/java/com/faforever/api/security/FafUserDetailsService.java

```diff
@@ -26,9 +26,9 @@ public FafUserDetailsService(UserRepository userRepository) {
   }
 
   @Override
-  public UserDetails loadUserByUsername(String username) throws UsernameNotFoundException {
-    User user = userRepository.findOneByLoginIgnoreCase(username)
-      .orElseThrow(() -> new UsernameNotFoundException("User could not be found: " + username));
+  public UserDetails loadUserByUsername(String usernameOrEmail) throws UsernameNotFoundException {
```

Comment:
Dont we still have accounts that have same email and username if compared to lowwer case how would that affect this code

Reply:
well good question, but I would argue that emails should be CI and username was searched CI even before. So if there are two accounts with same email, they will fail to log. Let's hope they will contact mod :)

Reply:
Ha interesting point: this is also an issue on password reset.
But I have checked, hibernate does not throw an Exception.
Two email accounts per user are not wanted, so we will ban of the two users and then need to scrap their email address I guess.